### PR TITLE
feat: add mdpresentation-viewer app to apps.json

### LINKED ISSUE #67: [Feature]: getting presentation viewer into the app store
### Description

check if all these points mentioned by Benedikt are covered before the next release: https://github.com/owncloud/web/issues/11330

- [x] manifest.json 
- [x] publicly available .zip file 
- [x] cover image
- [x] minimal QA on the app
- [x] add all information incl. screenshots to https://github.com/owncloud/awesome-ocis/blob/main/webApps/apps.json 

I think the first two points are already done for this app :)

### User Stories

As a creator of ocis web extensions
I want to my extension to be listed in the marketplace
So that users can find and install it easily

### Acceptance Criteria

all points above are implemented

### PR DESCRIPTION
**Pull request by saw-jan**
Added [mdpresentation-viewer](https://github.com/JankariTech/web-app-presentation-viewer) app.

- [x] cover.png
- [x] screenshots
- [x] add to apps.json

Closes https://github.com/JankariTech/web-app-presentation-viewer/issues/67

### CODE REVIEW

File: webApps/owncloud/web-extensions/mdpresentation-viewer/cover.png
Comment (saw-jan) [inline webApps/owncloud/web-extensions/mdpresentation-viewer/cover.png:1]: Should it be under `webApps/jankaritech/mdpresentation-viewer/cover.png` as mentioned in https://github.com/owncloud/awesome-ocis/pull/8#pullrequestreview-2243933463 ?

CC @kulmann

Comment (saw-jan) [reply webApps/owncloud/web-extensions/mdpresentation-viewer/cover.png:1]: Should `webApps/<org-name>/<repo-name>` be strict? if yes then the path would become `webApps/jankaritech/web-app-presentation-viewer/cover.png`

Comment (saw-jan) [reply webApps/owncloud/web-extensions/mdpresentation-viewer/cover.png:1]: moved to `webApps/jankaritech/web-app-presentation-viewer/`

**Comment by kulmann** [comment]
I didn't notice this PR, sorry 🙈  waiting for feedback about default CSP rules, see https://github.com/owncloud/ocis/pull/9993 - if that's getting included in the next ocis rolling release (tomorrow) we can merge here. Otherwise I'd like to add some description about the necessity to add a CSP rule.

File: webApps/apps.json [hunk 1]
```diff
@@ -198,6 +198,53 @@
...
+          "url": "https://github.com/JankariTech/web-app-presentation-viewer/releases/download/2.0.0/mdpresentation-viewer-2.0.0.zip"
+        },
+        {
+          "version": "1.0.0",
+          "url": "https://github.com/JankariTech/web-app-presentation-viewer/releases/download/1.0.0/mdpresentation-viewer-1.0.0.zip"
+        }
+      ],
+      "authors": [
+        {
+          "name": "Jankari Tech",
```
Comment (koebel) [inline webApps/apps.json]:
```suggestion
          "name": "JankariTech",
```

We spell the company name without space in the international context. Only in Nepal we have to add the space 🙈

**Review by kulmann — APPROVED** [review]
Thank you! 🥳

**Pull request merged**